forms-flow-api/src/formsflow_api/resources/roles.py

```diff
@@ -35,7 +36,7 @@ class KeycloakRolesResource(Resource):
     """Resource to manage keycloak list and create roles/groups."""
 
     @staticmethod
-    @auth.has_one_of_roles([ADMIN_GROUP, DESIGNER_GROUP])
+    @auth.has_one_of_roles([ADMIN_GROUP, DESIGNER_GROUP, REVIEWER_GROUP])

```

Comment:
this Designer group need access to this endpoint ?

Reply:
yes this endpoint is used in form edit by designer